.github/workflows/check-catalyst.yaml

```diff
@@ -14,10 +14,16 @@ jobs:
   constants:
     name: "Set build matrix"
     uses: ./.github/workflows/constants.yaml
+    with:
+      multiple_compilers: ${{ github.trigger == 'push' && github.ref_name == 'main' }}
```

Comment:
Nice! Minor but what do you think of updating the default to `gcc` that makes catalyst-check compatible with wheels?

Reply:
The default is gcc right now (because of the runner bug). Whether we gcc or clang as a default once that's fixed, I don't know 🤷

Reply:
Let's talk about it when it's fixed then :smile: